src/main/java/listeners/OnlinerTestListener.java

```diff
@@ -31,25 +39,66 @@ public void onTestFailedWithTimeout(ITestResult result) {
     @Override
     public void onStart(ITestContext context) {
         logger.info("The test class run started on " + context.getStartDate());
+        allureEnvironmentWriter(
+                ImmutableMap.<String, String>builder()
+                        .put("Browser", "Chrome")
+                        .put("Browser.Version", "96.0.4664.93 (Official build), (64 bit)")
+                        .build(), System.getProperty("user.dir")
+                        + "/target/allure-results/");
     }
 
     @Override
     public void onFinish(ITestContext context) {
         logger.info("The test class run finished on ... " + context.getEndDate());
     }
 
+    @Attachment(value = "Test.log", type = "text/plain")
+    public static byte[] appendLogToAllure() {
+        try {
+            return FileUtils.readFileToByteArray(new File("target/test.log"));
+        } catch (IOException ignored) {
+            return null;
+        }
+    }
+
+    public static void clearFile() throws IOException {
+        File myFoo = new File("target/test.log");
+        FileOutputStream fooStream = new FileOutputStream(myFoo, false);
+        byte[] myBytes = "".getBytes();
+        fooStream.write(myBytes);
+        fooStream.close();
+    }
+
     @Override
     public void onTestSuccess(ITestResult result) {
-        logger.info(result.getName() + " passed successfully...");
+        logger.info(result.getName() + " successful...");
+        try {
+            appendLogToAllure();
+            clearFile();
+        } catch (IOException e) {
+            e.printStackTrace();
```

Comment:
Nice idea to not throw an exception here because test execution should not be affected by actions with reporting. Even if actions with reporting fail test should keep going